src/sparseml/transformers/text_classification.py

```diff
@@ -592,14 +608,17 @@ def preprocess_function(examples):
                 range(data_args.max_predict_samples)
             )
 
-    # Log a few random samples from the training set:
-    if training_args.do_train:
-        for index in random.sample(range(len(train_dataset)), 3):
-            _LOGGER.info(f"Sample {index} of training set: {train_dataset[index]}.")
-
+    
+    
     # Get the metric function
-    if data_args.task_name is not None:
+    if data_args.task_name is not None and data_args.task_name not in ["biosses","pubmedqa","chemprot","DDI","GAD","hoc","bioasq"]:
```

Comment:
nit: any reason not to move the other `elif`s up and keep `elif data_args.task_name is not None` as the last/default elif so we don't need to maintain this list?